04-onion-routing.md

```diff
@@ -257,6 +257,9 @@ This is a more flexible format, which avoids the redundant `short_channel_id` fi
     1. type: 6 (`short_channel_id`)
     2. data:
         * [`short_channel_id`:`short_channel_id`]
+    1. type: 8 (`basic_mpp`)
+	2. data:
+	    * [`tu64`:`total_msat`]
```

Comment:
I suggest a minimal two-tuple of `(amt, randID)` as mentioned in my earlier comment.

Reply:
Shouldn't `randID` be defined as a separate type, so that it can be used to pay non-amp to zero-valued invoices?